buildbot/README.md

```diff
@@ -66,7 +64,8 @@ If the directory contains data files with `.data` extension, they'll be copied o
 
 You can also specify job configuration options as further POST parameters:
 
-- `skipseashell`, which lets you supply plain HLS C code as input.
+**NOTE(rachit)**: These options are out of date. Document them properly here.
```

Comment:
I think they’re all right! We’re missing a few, but the ones that are here look fine to me.